stestr/subunit_trace.py

```diff
@@ -376,7 +377,7 @@ def trace(stdin, stdout, print_failures=False, failonly=False,
     elapsed_time = stop_time - start_time
 
     if count_tests('status', '.*') == 0:
-        print("The test run didn't actually run any tests")
+        print("The test run didn't actually run any tests", file=sys.stderr)
```

Comment:
This is the only place I'm unsure of. (and below) The way this is handled it's the normal flow for subunit-trace and ran through the stream and no test cases were encountered. I'm not sure if this should go to stderr since normal subunit-trace output all goes to stdout.